experimenter/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.tsx

```diff
@@ -342,6 +349,92 @@ export const FormAudience = ({
     setPopulationPercent(populationPercent);
   };
 
+  const getTargetPopulationSize = (sizingData: SizingTarget) => {
+    const firstSizingKey = Object.keys(sizingData.sample_sizes)[0];
+    const firstSizingMetrics = sizingData.sample_sizes[firstSizingKey].metrics;
+    const firstMetricKey = Object.keys(firstSizingMetrics)[0];
+
+    return firstSizingMetrics[firstMetricKey].number_of_clients_targeted;
+  };
+
+  const buildSizingKey = (
+    appId: string | undefined,
+    channel: string | undefined,
+    locales: string[] | undefined,
+    countries: string[] | undefined,
+  ): string | null => {
+    if (
+      !(
+        appId &&
+        channel &&
+        locales &&
+        locales.length > 0 &&
+        countries &&
+        countries.length > 0
+      )
```

Comment:
so this will only work if we have locale AND country set?

Reply:
(e.g. won't work if I only set country and no locale)

Reply:
Desktop uses locales and mobile uses languages, does it mean it is not for mobile experiments? 🤔